%YAML:1.0
---
CameraMat: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.9310943000000000e+03, 0., 9.4977191700000003e+02, 0.,
       1.9323061980000000e+03, 5.5201076200000000e+02, 0., 0., 1. ]
DistCoeff: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -5.7995600000000003e-01, 3.7999699999999997e-01,
       -1.7370000000000000e-03, 3.0929999999999998e-03,
       -1.9015499999999999e-01 ]
ImageSize: [ 1920, 1020 ]
CameraExtrinsicMat: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 9.9995236083024763e-01, -9.1760100055196230e-03,
       -3.3281992718133093e-03, 3.4753286545720921e-02,
       -3.3699746291749265e-03, -4.5409745827154841e-03,
       -9.9998401128259939e-01, -1.8681017761246443e-01,
       9.1607500245892634e-03, 9.9994758882164303e-01,
       -4.5716811753721527e-03, -1.6901053236355607e-01, 0., 0., 0., 1. ]
Pose: [ 3.4753286545720921e-02, -1.8681017761246443e-01,
    -1.6901053236355607e-01, 1.5753680492158391e+00,
    -3.3282054162047871e-03, 9.1761896026029306e-03 ]
